api/app/controllers/spree/api/v2/platform/google_export_options_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Spree
+  module Api
+    module V2
+      module Platform
+        class GoogleExportOptionsController < ResourceController
+          def show
```

Comment:
The `show` method for a resource should return information regarding the resource itself (in this case, the resource is an options object). In this case, I would rename this method to be something like `export_rss_feed` since we are not returning information about the options object.